x-pack/plugins/cloud_security_posture/server/routes/compliance_dashboard/get_grouped_findings_evaluation.ts

```diff
@@ -20,6 +21,28 @@ export interface FailedFindingsQueryResult {
   aggs_by_resource_type: Aggregation<FailedFindingsBucket>;
 }
 
+export interface BenchmarkVersionQueryResult extends KeyDocCount, FailedFindingsQueryResult {
+  failed_findings: {
+    doc_count: number;
+  };
+  passed_findings: {
+    doc_count: number;
+  };
+  asset_count: {
+    value: number;
+  };
+  aggs_by_benchmark_name: Aggregation<KeyDocCount>;
+}
+export interface FailedFindingsBucket extends KeyDocCount {
```

Comment:
Could you please rename the interface name? The bucket not only includes failed findings.